models/api-models.js

```diff
@@ -0,0 +1,7 @@
+const fs = require("fs/promises")
+
+exports.readApiDetails = () => {
+    return fs.readFile(`${__dirname}/../endpoints.json`, "utf-8").then((data) => {
+        return JSON.parse(data)
+    })
```

Comment:
We don't actually need a model in this case, take a look at my comment on your controller for further info! 😄